entity-framework/efcore-and-ef6/index.md

```diff
@@ -1,77 +1,74 @@
 ---
-title: Compare Entity Framework 6 and Entity Framework Core - EF
-description: Provides guidance on how to choose between Entity Framework 6 and Entity Framework Core.
-author: rowanmiller
-ms.date: 10/27/2016
+title: Compare EF6 and EF Core
+description: Guidance on how to choose between EF6 and EF Core.
+author: ajcvickers
+ms.date: 01/23/2019
 ms.assetid: a6b9cd22-6803-4c6c-a4d4-21147c0a81cb
 uid: efcore-and-ef6/index
 ---
 
 # Compare EF Core & EF6
 
-Entity Framework is an object-relational mapper (O/RM) for .NET. This article compares the two versions: Entity Framework 6 and Entity Framework Core.
+## EF Core
 
-## Entity Framework 6
+Entity Framework Core ([EF Core](../core/index.md)) is a modern object-database mapper for .NET. It supports LINQ queries, change tracking, updates, and schema migrations.
 
-Entity Framework 6 (EF6) is a tried and tested data access technology. It was first released in 2008, as part of .NET Framework 3.5 SP1 and Visual Studio 2008 SP1. Starting with the 4.1 release it has shipped as the [EntityFramework](https://www.nuget.org/packages/EntityFramework/) NuGet package. EF6 runs on the .NET Framework 4.x and .NET Core from 3.0 onwards.
+EF Core works with SQL Server/SQL Azure, SQLite, Azure Cosmos, MySQL, ProstgreSQL, and many more databases through a [database provider plugin model](../core/providers/index.md).
```

Comment:
"Azure SQL Database" and "Azure Cosmos DB" seem to be the official short product names

Reply:
Thanks @ErikEJ!